### PR TITLE
Up and running with Vagrant, Update Pig and ES, cleanup

### PR DESCRIPTION
**Pull request by jatrost**

### CODE REVIEW

File: binarypig/pom.xml [hunk 1]
```diff
@@ -37,7 +37,7 @@
     <dependency>
       <groupId>org.apache.pig</groupId>
       <artifactId>pig</artifactId>
-      <version>0.8.1</version>
+      <version>0.12.1</version>
```
Comment (telvis07) [inline binarypig/pom.xml:40]: nice!

**Comment by telvis07** [comment]
lgtm

File: binarypig/pom.xml (hunk 1 above)
Comment (ekamioka) [inline binarypig/pom.xml:40]: Nice...tks!

**Comment by phreakinggeek** [comment]
I gave it a shot using vagrant and everything i've tested has been working well

File: binarypig/pom.xml [hunk 2]
```diff
@@ -53,7 +53,7 @@
     <dependency>
       <groupId>org.apache.hadoop</groupId>
       <artifactId>hadoop-core</artifactId>
-      <version>0.20.2-cdh3u3</version>
+      <version>1.2.1</version>
```
Comment (crohling) [inline binarypig/pom.xml:56]: You mean you DONT want to use ancient versions of Hadoop? Inconceivable. +1

File: install-ubuntu.sh [hunk 3]
```diff
@@ -0,0 +1,231 @@
...
+  <value>/var/hadoop/hadoop-\${user.name}</value>
+  </property>
+</configuration>
+EOF
+
+cat > /etc/hadoop/hdfs-site.xml <<EOF
+<configuration>
+  <property>
+  <name>dfs.replication</name>
+  <value>1</value>
```
Comment (crohling) [inline install-ubuntu.sh]: Is this recommended for just a test env? Is that all vagrant is being used for?

File: install-ubuntu.sh [hunk 4]
```diff
@@ -0,0 +1,231 @@
...
+<configuration>
+  <property>
+  <name>mapred.job.tracker</name>
+  <value>localhost:9001</value>
+  </property>
+</configuration>
+EOF
+
+hadoop namenode -format
+start-all.sh
```
Comment (crohling) [inline install-ubuntu.sh]: Is start-all.sh on the path?

File: install-ubuntu.sh (hunk 3 above)
Comment (jatrost) [inline install-ubuntu.sh]: Yeah this is just for the vagrant setup.  I figured anyone running this on a cluster could use this script as a guide, but not follow it verbatim.

File: install-ubuntu.sh (hunk 4 above)
Comment (jatrost) [inline install-ubuntu.sh]: correct, after installing the hadoop debian package start-all is on the path.

**Comment by jatrost** [comment]
@crohling I renamed `install-ubuntu.sh` to `install-ubuntu-vagrant.sh` so it is more clear that this script is for testing with Vagrant.

**Pull request merged**